compiler/rustc_expand/src/mbe/macro_parser.rs

```diff
@@ -70,21 +70,20 @@
 //! eof: [a $( a )* a b ·]
 //! ```
 
+use rustc_errors::ErrorGuaranteed;
```

Comment:
Makes sense to move this `use` down with the other ones, below the `pub(crate)` ones.